src/aequitas/plotting.py

```diff
@@ -458,9 +499,46 @@ def plot_disparity(self, disparity_table, group_metric, attribute_name,
         else:
             labels = sorted_df['attribute_value'].values
 
+        # if df includes significance columns, add stars to indicate significance
+        if len(list(sorted_df.columns[sorted_df.columns.str.contains('_significance')])) > 0:
```

Comment:
I'm pretty sure there's a Pandas "count" method you could use here....

Reply:
good call! going with 
`if sorted_df.columns[
            sorted_df.columns.str.contains('_significance')].value_counts().sum() > 0:`